docs/framework/data/adonet/sql/sql-server-security.md

```diff
@@ -41,12 +41,9 @@ ms.workload:
  Describes security considerations for [!INCLUDE[ssNoVersion](../../../../../includes/ssnoversion-md.md)] Express.  
   
 ## Related Sections  
- [Security and Protection (Database Engine)](http://msdn2.microsoft.com/library/bb510589\(SQL.100\).aspx.)  
- [!INCLUDE[ssNoVersion](../../../../../includes/ssnoversion-md.md)] Books Online security topics.  
-  
- [Security Considerations for SQL Server](http://go.microsoft.com/fwlink/?LinkId=98587)  
- [!INCLUDE[ssNoVersion](../../../../../includes/ssnoversion-md.md)] Books Online security topics.  
-  
+[Security Center for SQL Server Database Engine and Azure SQL Database](https://docs.microsoft.com/sql/relational-databases/security/security-center-for-sql-server-database-engine-and-azure-sql-database)  
```

Comment:
In order for these links to work in the offline books you need to remove the docs.microsoft.com from the link.

So the link would look like (/sql/relational-databases/security/security-center-for-sql-server-database-engine-and-azure-sql-database)

But I need to file a feature request for a better way to doing these links

Reply:
@mairaw Thanks for the guidance. I've committed a new version with the fix. I don't need the ".md" file extension, do I?